### PR TITLE
fix CJS default export in types

### PR DESCRIPTION
**Pull request by FabianFrank**
fixes https://github.com/pinojs/pino-abstract-transport/issues/71

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -80,10 +69,24 @@ declare function build(
  * @returns {Transform} the wrapped split2 instance
  */
 declare function build(
-  fn: (transform: Transform & OnUnknown) => Transform & OnUnknown,
+  fn: (transform: Transform & build.OnUnknown) => Transform & build.OnUnknown,
   opts: EnablePipelining
 ): Transform;
 
-export { OnUnknown };
+declare namespace build {
```
Comment (FabianFrank) [inline index.d.ts:76]: Declaring a namespace and merging the interface onto it is what prevents this from making a backwards breaking change, i.e. `import build, { OnUnknown } from "pino-abstract-transport"` still works.

**Comment by FabianFrank** [comment]
FYI the `npm install` error in `CI / Test (14, windows-latest)` during  appears unrelated to my changes:
```
npm ERR! code EPERM
npm ERR! syscall rename
npm ERR! path D:\a\pino-abstract-transport\pino-abstract-transport\node_modules\tap\node_modules\@jridgewell\.trace-mapping.DELETE\node_modules\@jridgewell
npm ERR! dest D:\a\pino-abstract-transport\pino-abstract-transport\node_modules\tap\node_modules\@jridgewell\trace-mapping\node_modules\@jridgewell
npm ERR! errno -4048
```
Maybe just requires a re-run?

**Comment by mcollina** [comment]
Could you add an exclude in the actions for windows/node v14? It's broken in gha beyond repair.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**